atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -130,7 +131,7 @@ protected TransactionManager createManager() {
 
     @Override
     protected Transaction startTransaction() {
-        return startTransactionWithOptions(new TransactionOptions());
+        return Iterables.getOnlyElement(txMgr.startTransactions(List.of(PreCommitConditions.NO_OP)));
```

Comment:
I get that you're making this class less terrible, but is it directly tied to the other changes? Maybe it is (because we're now _actually_ using a serializable table), but just wanted to check.

Reply:
Yes, it's necessary (otherwise we get a load of lock watch-related failures).